magichome/magichome.js

```diff
@@ -190,17 +168,16 @@ module.exports = function (RED) {
         }
 
         let internalState = new LampState();
-
         let node = this;
         let light = new MagicHomeControl(host);
 
         function setState(value) {
 
-            if (value === internalState.C_ON || value === true) {
+            if (value || value === internalState.C_ON) {
                 light.turnOn(function () {
                     queryLampState();
                 });
-            } else if (value === internalState.C_OFF || value === false) {
+            } else if (!value || value === internalState.C_OFF) {
```

Comment:
falsy makes it prone to unexpected behaviour. msg.payload.power = "" would turn the light off